### DISCUSSION TITLE
Update soon?

### DISCUSSION THREAD

**Discussion by marlonjames71** [notaapp/talk#321] [Random]
Was just curious as I thought I read there'd be an update at the beginning of Sep.

**Comment by astoilkov** [comment]
Hey. Yeah, we will have a new update in 1-2 weeks. We thought we will be ready earlier but couldn't figure out the UX improvements for the tabs.

**Reply by marlonjames71** [reply]
Ahhh no worries. Thanks! Looking forward to the update.

**Reply by webwesen** [reply]
checking in: 2022-11-21
an update on ETA might be helpful... thanks!

**Reply by marlonjames71** [reply]
Been wondering where the update is as well.

**Reply by astoilkov** [reply]
Sorry for the delay. Both me and @erusev are working on the release every day.

The reason we are delaying the release is that we thought we figured out the new way tabs should work, but after a few weeks of testing, we realized we were wrong. We are now working on a new way tabs will function.

Also, at the same time, we are transitioning to a new technology that will allow us to iterate faster on products. However, this is a big task that costs us a lot of time.

After this release is done we should go back to a normal schedule of publishing releases.

ETA is around 2-3 weeks. I failed with the estimation last time I hope I don't know 😉.

**Reply by borekb** [reply]
Looking forward to a 🎄 present 😄. Good luck!

**Reply by srid** [reply]
> we are transitioning to a new technology 

Could you share with us what this new technology is? Nota is currently swift and fast; will it remain so in future?

**Reply by astoilkov** [reply]
These are internal ones. It shouldn't affect users too much. We are moving from Webpack to Vite and from CSS files to Tailwind.

BTW, Nota isn't using Swift, it's using Electron. There are quite a few people who think Nota is a native app — see this comment https://github.com/notaapp/talk/discussions/269#discussioncomment-3204456 that comes from [a conversation with Sindre Sorhus](https://gist.github.com/sindresorhus/be6a363ea0bda1ce7e478785e35c6fb3#:~:text=I%20totally%20agree%20that%20most%20Electron%20apps%20are%20shitty.%20It%27s%20interesting%20that%20because%20our%20app%20is%20Electron%20some%20people%20immediately%20label%20it%20shitty.%20Another%20interesting%20thing%20is%20that%20we%20have%20some%20users%20sending%20us%20feedback%20thinking%20the%20app%20is%20native.).

**Reply by GraphicOwls** [reply]
That's nice you guys are moving to Tailwind! Love TW. Are you moving to Vite for faster compiling and performance?

**Reply by astoilkov** [reply]
Yeah, Tailwind is awesome (probably the future).

> Are you moving to Vite for faster compiling and performance?

Yeah. It's a lot faster. Development has been a lot more pleasant with it. Also, Webpack doesn't seem to have any future so it's a necessary shift at should have happened.

**Comment by erusev** [comment]
We'll need a little more time, but the update will include some additional improvements (ex: in search). We're aiming at Jan 16 for the release date.

**Reply by marlonjames71** [reply]
Not happening today either?

**Reply by erusev** [reply]
It's mostly done, but we need a few more days to test it 😔 It should be out later this week and will be based on a more productive tech stack, so future updates should be faster.

**Reply by marlonjames71** [reply]
Awesome! Looking forward to it! Are you guys pumped?

**Reply by erusev** [reply]
Well, the new codebase is much more fun to work with, so if pumped means excited, then sure 😊

**Reply by marlonjames71** [reply]
Well that's good. I also meant pumped for others to check it out

**Reply by mourner** [reply]
@erusev just curious, what's the new tech stack? Hoping for something like Tauri in place of Electron 🤞

**Comment by lzilioli** [comment]
Any updates on the release date?

**Reply by erusev** [reply]
Yes, we're planning the release for the 25th, which is in a couple of days. Thanks for the patience 🙏

**Comment by erusev** [comment]
0.39 is out now ✨ 

`Nota` → `Check for Updates` to get it.

<img width="211" alt="CleanShot 2023-01-26 at 16 22 57@2x" src="https://user-images.githubusercontent.com/184170/214859974-0e374117-6a53-46fe-83b9-28a74203f354.png">

Thanks for the patience, and as usual, feedback is very much welcome 🙏

**Reply by GraphicOwls** [reply]
Awesome!!! Thank you 🙏

**Reply by harveyr** [reply]
Congratulations and thank you! Nota is amazing! ❤️

**Reply by marlonjames71** [reply]
Double space to add a period!!! Woohoo! Glad it's here finally

**Reply by marlonjames71** [reply]
if you're looking for feedback, do you have a change log of some kind so we can keep a special eye for certain things? This would probably be helpful.

**Reply by GraphicOwls** [reply]
https://nota.md/changelog.html

**Reply by marlonjames71** [reply]
Dang! That's a lot of solid improvements!
